slack_wrapper.gs

```diff
@@ -1,16 +1,31 @@
+class SlackEventBuilder {
```

Comment:
This doesn't need to be a class: a simple function would do. There is no state to encapsulate and no inheritance.  

Also, returning from a constructor like this is a bit weird - I had no idea you could do it in javascript. It feels like if you have a statement like:
```
var a = new ClassBuilder(args);
```

then `a` should be of type `ClassBuilder`, which isn't the case here.

Reply:
changed to function in new commit